docs/components/DocumentationSidebarLink.tsx

```diff
@@ -77,23 +77,23 @@ export default class DocumentationSidebarLink extends React.Component<Props> {
   }
 
   isSelected() {
-    if (!this.props.url) {
+    if (!this.props.router) {
       console.log('[debug] isSelected bailed out, no url', this.props);
```

Comment:
```suggestion
      console.log('[debug] isSelected bailed out, no router', this.props);
```
Just spotted this right now, this suggestion is not merge blocking in any way. 🙂

Reply:
ooh nice, thanks!

Reply:
I don't think I have ever seen this log statement though 🙈  There is always a pathname, even if its `https://docs.expo.dev` -> `/`

Reply:
Maybe we can just remove this log then?